src/platform/plugins/shared/share/public/components/export_integrations/export_integrations.tsx

```diff
@@ -157,8 +166,8 @@ function ManagedFlyout({
           </h2>
         </EuiTitle>
       </EuiFlyoutHeader>
-      <EuiFlyoutBody>
-        <EuiFlexGroup direction="column">
+      <EuiFlyoutBody css={flyoutBodyCss}>
+        <EuiFlexGroup css={{ height: '100%' }} direction="column">
```

Comment:
I see this causing a big gap of empty space in the middle of the body of the PNG/PDF export flyouts. I don't think we want this. 

<img width="1902" height="2054" alt="image" src="https://github.com/user-attachments/assets/f8d92d0d-d62f-44c7-9919-2b67d5094a8e" />